mautic/mautic
# implement code review suggestions from PR #11971

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR is to address the code review from [TPROD-312/TPROD-358-corebundle-theme-code-review](https://github.com/mautic/mautic/pull/11971)



#### Steps to test this PR:
Compare the code changes with the review from [TPROD-312/TPROD-358-corebundle-theme-code-review](https://github.com/mautic/mautic/pull/11971)


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.
